testsuite/features/secondary/buildhost_docker_build_image.feature

```diff
@@ -61,18 +61,23 @@ Feature: Build container images
     And I wait at most 60 seconds until all "3" container images are built correctly on the Image List page
     # We should see the same result via API.
     # Also, check that all inspect actions are finished:
-    And I wait at most 660 seconds until image "suse_key" with version "latest" is built and inspected successfully via API
-    And I wait at most 660 seconds until image "suse_simple" with version "latest" is built and inspected successfully via API
-    And I wait at most 660 seconds until image "suse_real_key" with version "latest" is built and inspected successfully via API
+    And I wait at most 600 seconds until image "suse_key" with version "latest" is built successfully via API
+    And I wait at most 300 seconds until image "suse_key" with version "latest" is inspected successfully via API
+    And I wait at most 600 seconds until image "suse_simple" with version "latest" is built successfully via API
+    And I wait at most 300 seconds until image "suse_simple" with version "latest" is inspected successfully via API
+    And I wait at most 600 seconds until image "suse_real_key" with version "latest" is built successfully via API
+    And I wait at most 300 seconds until image "suse_real_key" with version "latest" is inspected successfully via API
     Then the list of packages of image "suse_key" with version "latest" is not empty
     And the list of packages of image "suse_simple" with version "latest" is not empty
     And the list of packages of image "suse_real_key" with version "latest" is not empty
 
   Scenario: Build same images with different versions
     When I schedule the build of image "suse_key" with version "Latest_key-activation1" via API calls
     And I schedule the build of image "suse_simple" with version "Latest_simple" via API calls
-    And I wait at most 660 seconds until image "suse_simple" with version "Latest_simple" is built and inspected successfully via API
-    And I wait at most 660 seconds until image "suse_key" with version "Latest_key-activation1" is built and inspected successfully via API
+    And I wait at most 600 seconds until image "suse_simple" with version "Latest_simple" is built successfully via API
```

Comment:
I would suggest the following change of order:
1. suse_key built
2. suse_simple build
3. suse_key inspected
4. suse_simple inspected

as it might bring less waiting, and it's somehow more logical

Reply:
I would prefer two different scenarios following the whole workflow for each of the images. So a scenario for each image.